index.html

```diff
@@ -164,9 +164,6 @@ <h3>For argon2</h3>
                 var optionsLink                   = document.querySelector('a.options');
                 var optionsDiv                    = document.querySelector('div.options');
 
-                urlInput.value                    = window.location.protocol + '//' + window.location.host;

```

Comment:
Ok but I think we should stop the execution if the URL is empty and show an error message in the index page or the addon...
The current version of the addon doesn’t allow an empty URL (or key) which I think is correct.

Reply:
Yes, I noticed that there is no control on that and I planned to add it.

I passed some time looking if there is not a way to read the clipboard to see if there is an url inside but I think that there is no way to do it (I think for security reasons) 😐

Reply:
Thanks for thinking to add it. It would be nice I think.
Yes, I think it’s better to not read the clipboard. It can lead to a security problem.

Reply:
>It would be nice I think.

done ;)